src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -169,6 +170,25 @@ export const selectTxIdsByFilter = createCachedSelector(
     : 'txIdsByFilter',
 )
 
+export const selectTxIdsByFilterWithPendingFirst = createCachedSelector(
+  selectTxs,
+  selectTxIdsByFilter,
+  (txs, filteredTxIds): TxId[] => {
+    return [...filteredTxIds].sort((a, b) => {
+      if (txs[a].status === TxStatus.Pending && txs[b].status !== TxStatus.Pending) return -1
+      if (txs[b].status === TxStatus.Pending && txs[a].status !== TxStatus.Pending) return 1
+
+      return 0
+    })
+  },
+)((_state: ReduxState, filter) =>
+  filter
+    ? `${filter.accountId ?? 'accountId'}-${filter.txStatus ?? 'txStatus'}-${
+        filter.assetId ?? 'assetId'
+      }`
+    : 'txIdsByFilterWithPendingFirst',
+)
+
```

Comment:
Tbh I think we shouldn't really care about this one, we could simply use `selectTxIdsByFilter` and let pending TXs being a bit under, people are not spamming TXs at this point

Reply:
Fine with this one in the interim!